packages/@ama-sdk/showcase-sdk/project.json

```diff
@@ -7,10 +7,12 @@
   "targets": {
     "build": {
       "executor": "nx:noop",
+      "inputs": ["default", "^default", "^cli"],
```

Comment:
not needed

Reply:
It was a tentative to fix the cache because the showcase-sdk build was not invalidated and thus not rebuild in the CI.
Anyway it does not work so it should be handled in another task

Reply:
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/9dd07ec6-9dbd-4709-b13a-beb277c6d1e2)
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/a67523b9-064f-4ee3-8daf-1b3a4a5343d0)